src/main/java/io/anserini/index/IndexReaderUtils.java

```diff
@@ -447,6 +447,31 @@ public static Map<String, Long> getDocumentVector(IndexReader reader, String doc
     return docVector;
   }
 
+  /**
+   * Returns the dense vector representation of a document, if available.
+   *
+   * @param reader index reader
+   * @param docid document id
+   * @return vector as float array, or null if not available
+   * @throws IOException if error encountered during access to index
+   */
+  public static float[] getDenseVector(IndexReader reader, String docid) throws IOException {
```

Comment:
Do we need this method anymore? I don't think we ever parse from string in the current impl?